ui/app/AppLayouts/Profile/popups/ChatLinksPreviewModal.qml

```diff
@@ -203,7 +203,7 @@ ModalPopup {
             }
         }
 
-        StyledText {
+        StatusBaseText {
```

Comment:
All of the `StatusBaseText` usages here need a `color: Theme.palette.directColor1`, otherwise they won't work across themes.

Reply:
Thanks for pointing out. Fixed it :)